tests/datasets/cdc/test_groupassignee.py

```diff
@@ -0,0 +1,221 @@
+from datetime import datetime
```

Comment:
This file was moved from `tests/datasets/test_groupassignee.py` but was so substantially changed by the reformat that it's being displayed as new. This is largely functionally unchanged. The fixtures are now the `Event` subtypes (as they would be after being parsed from the Kafka payload) and the message filter tests were removed as the table filter is now tested on it's own.